modules/internal/String.chpl

```diff
@@ -128,12 +154,30 @@ module String {
       }
     }
 
+    /*
+      Construct a new string from ``cs``. If ``owned`` is set to true, the
```

Comment:
I'd be tempted to state the type of "cs" here and "s" in the previous constructor.  The reader can figure it out, but it would make differentiating the two constructors quicker